flytekit/types/schema/types.py

```diff
@@ -219,10 +222,10 @@ def format(cls) -> SchemaFormat:
 
     def __init__(
         self,
+        downloader: typing.Callable[[str, os.PathLike], None],
```

Comment:
Could we keep the `downloader` default value as None?

Reply:
Sure, but "None" is not callable. Should I add some checks before using it?

Reply:
yes, we need it